README.md

```diff
@@ -221,6 +221,21 @@ convert it to your enum type and construct the object with it.
 Custom type definitions are also validated against a [schema](/schema/custom-codec-schema.json). See the [Schema Validation](#schema-validation) 
 section for details of the validation.
 
+### A Reference Mapping to Versions
+
+|Client Protocol Version|Hazelcast Version|
+|-----|-----|
+|2.0|4.0|
+|2.0.1|4.0.1|
+|2.1|4.1|
+|2.2|4.2|
+|2.3|5.0|
+|2.4|5.1|
+|2.5|5.2|
+|2.6|5.3|
+|2.7|5.4|
+
+
```

Comment:
If we are using hazelcast version now, why do we have this table? I think we can't change the older released code, that's why? Similarly, why do we have new/protocol-definitions and protocol-definitions?